indy-besu/smart_contracts/package.json

```diff
@@ -9,6 +9,9 @@
     "ts-lint": "tslint --fix -p tsconfig.json -c tslint.json",
     "solhint": "solhint --fix -f unix -c solhint.json contracts/**/*.sol",
     "lint": "yarn ts-lint && yarn solhint",
+    "ts-lint-check": "tslint -p tsconfig.json -c tslint.json",
+    "solhint-check": "solhint -c solhint.json contracts/**/*.sol",
+    "lint-check": "npm run ts-lint-check && npm run solhint-check",
```

Comment:
Why did we use npm instead of yarn here?

Reply:
Changed to yarn